bip-taproot.mediawiki

```diff
@@ -236,6 +236,22 @@ def taproot_output_script(internal_pubkey, script_tree):
 
 [[File:bip-taproot/tree.png|frame|This diagram shows the hashing structure to obtain the tweak from an internal key ''P'' and a Merkle tree consisting of 5 script leaves. ''A'', ''B'', ''C'' and ''E'' are ''TapLeaf'' hashes similar to ''D'' and ''AB'' is a ''TapBranch'' hash. Note that when ''CDE'' is computed ''E'' is hashed first because ''E'' is less than ''CD''.]]
 
+To spend this output using script ''D'', the control block would contain the following data in series:
```

Comment:
"In series" sounds weird to me, is that a common expression in English?

Reply:
I was trying to sound more computational :-)
"...in this order" would be precisely what I mean though.

Reply:
But do you think it's clear? I'd believe you if you think so, but to me it's just confusing.

Reply:
Updated. I think it's more clear now, thanks.

"To spend this output using script ''D'', the control block would contain the following data in this order:"